### DISCUSSION TITLE
How to run frontend ?

### DISCUSSION THREAD

**Discussion by atiq-inument** [notifo-io/notifo#252] [Q&A]
This project is really great. 

I cloned the repo, installed MongoDB, and then ran the backend app. 
It is running, but I am stuck on the frontend. 
Can you please guide me on how to run the frontend app?

**Comment by SebastianStehle** [comment]
Hi,

thank you for your nice words. In dev mode you have to run the frontend independently. 

Just go to the frontend folder and run

```
npm i
npm run dev
```

It is using vite, btw. The frontend is proxied by the backend, so you go the backend URL (e.g. https://localhost:5001), not the frontend URL
